Packs/PrismaSaasSecurity/Integrations/SaasSecurity/SaasSecurity_test.py

```diff
@@ -0,0 +1,277 @@
+import io
+from freezegun import freeze_time
+import pytest
+
+from SaasSecurity import Client
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
```

Comment:
am I missing something? I think you are not using this import

Reply:
Just the DemistoException. i'll import that particular class.